magic-link/magic-link-templates.php

```diff
@@ -252,6 +130,21 @@ public function __construct( $template = null ) {
             return;
         }
 
+        /**
+         * Attempt to load sooner, rather than later; corresponding post record details.
+         */
+
+        $this->post = $this->fetch_post_by_incoming_hash_key();
```

Comment:
@kodinkat since the post_id is available in the $this->parts. Would that be simpler?